facturacion_mexico/facturacion_fiscal/doctype/configuracion_reclasificacion_fiscal_mexico/configuracion_reclasificacion_fiscal_mexico.py

```
@@ -0,0 +1,174 @@
+					"cuenta_destino": cuenta_destino,
+					"mrfpe_ref": mrfpe_ref,
+					"nota": row.rol_fiscal,
+				},
+			)
+
+		self.ultima_deteccion = now_datetime()
+		self.save()
+		return {"message": _("{0} cuentas cargadas.").format(len(self.reglas))}
+
+	@frappe.whitelist()
+	def aplicar(self):
```

Ruff (0.15.12)

[warning] Too many branches (14 > 12)

(PLR0912)